core/migrations/0098_remove_resourcerequest_membership.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-11-15 19:42
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0097_userprofile_guacamole_color'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='resourcerequest',
+            name='membership',
+        ),
+    ]
```

Comment:
Reminder to whomever merges the PR that migrations are included, this is the third `0098` active PR 😉

Reply:
Oh boy. Thanks for the warning. :)